.github/workflows/dockerfile-builder.yml

```diff
@@ -0,0 +1,41 @@
+name: Dockerfile Builder
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
How about limit it by path of projects with docker file? Or is it cheap to run this workflow on every push to master?

Reply:
I updated the PR, please have a look again. I am filtering the workflow on a job level though so I can combine it easily with the GH build matrices.